muttlib/dbconn/base.py

```diff
@@ -126,11 +126,14 @@ def __setattr__(self, attr, value):
                 elif attr == 'driver':
                     value = format_drivername(self.dialect, value)
                 attr = 'drivername'
+            self.conn_url = self.conn_url.set(**{attr: value})
```

Comment:
https://docs.sqlalchemy.org/en/20/changelog/migration_14.html#change-5526

Reply:
I don't quickly understand the change, mind explaining it to me if you remember? :)

If it's a breaking change with sqlalchemy 1.3 we could add a version check to remain compatible with both 1.3 and 1.4.